### DISCUSSION TITLE
MariaDB Quickstart

### DISCUSSION THREAD

**Discussion by giscus** [appscode/community#72] [KubeDB]
# MariaDB Quickstart

KubeDB simplifies Provisioning, Upgrading, Scaling, Volume Expansion, Monitor, Backup, Restore for various Databases in Kubernetes on any Public & Private Cloud

https://kubedb.com/docs/v2024.11.18/guides/mariadb/quickstart/overview/

**Comment by doteater** [comment]
In the example where we demonstrate the deletion behavior, shouldn't we show that there are no longer petsets, no statefulsets? (example: when displaying the output of "kubectl get sts,svc,secret,pvc -n demo"). Assuming you used to use statefulsets and this is an artifact, was a tiny bit confusing running through this.
